app/src/main/java/io/gnosis/safe/ui/dialogs/UnstoppableInputDialog.kt

```diff
@@ -58,6 +61,15 @@ class UnstoppableInputDialog : BaseViewBindingDialogFragment<DialogUnstoppableIn
             backButton.setOnClickListener { dismiss() }
             confirmButton.setOnClickListener { onClick.offer(Unit) }
             dialogUnstoppableInputDomain.showKeyboardForView()
+            chainRibbon.text = selectedChain.name
+            try {
+                chainRibbon.setTextColor(Color.parseColor(selectedChain.textColor))
```

Comment:
we should introduce extension function and add it to AppUtils. This code is reused in many places